test/integration/lerna-exec.test.js

```diff
@@ -104,4 +104,48 @@ describe("lerna exec", () => {
     expect(stdout).toMatch("package-2: file-2.js");
     expect(stdout).toMatch("package-2: package.json");
   });
+
+  test.concurrent("--bail=false <cmd>", async () => {
+    const cwd = await initFixture("ExecCommand/basic");
+    const args = [
+      "exec",
+      "--bail=false",
+      "--concurrency=1",
+      "--",
+      "npm run fail-or-succeed",
+    ];
+
+    const { stdout, stderr } = await execa(LERNA_BIN, args, { cwd });
+    expect(stderr).toMatch(
+      "Failed at the package-1@1.0.0 fail-or-succeed script 'echo \"failure!\" && exit 1'."
+    );
+    expect(stdout).toMatch("failure!");
+    expect(stdout).toMatch("success!");
+  });
+
+  test.concurrent("--bail=true <cmd>", async () => {
```

Comment:
@evocateur apologies, but what do you mean by `--no-bail`? The `bail=false` test case is above on line 108, but I assume you mean something else. This is my first time diving into the Lerna codebase, so I'm probably just missing something :)